src/config/category.js

```diff
@@ -0,0 +1,13 @@
+const categories = {
```

Comment:
ideally categories should have been an array instead of object with properties

Reply:
yes !! resolved